docs/views/expander.md

```diff
@@ -0,0 +1,335 @@
+
+![Screenshot of an Expander icon in expand state, on iOS and Android](expander-images/icon-collapse.png "Expand icon on iOS and Android")
+
+For more information about triggers, see [Xamarin.Forms Triggers](/xamarin/xamarin-forms/app-fundamentals/triggers.md).
+
+## Embed an Expander in an Expander
+
+The content of an `Expander` can be set to another `Expander` control, to enable multiple levels of expansion. The following XAML shows an `Expander` whose content is another `Expander` object:
+
+```xaml
+<Expander>
+    <Expander.Header>
+        <Label Text="{Binding Name}"
+               FontAttributes="Bold"
+               FontSize="Medium" />
+    </Expander.Header>
+    <Expander Padding="10">
+        <Expander.Header>
+            <Label Text="{Binding Location}"
+                   FontSize="Medium" />
+        </Expander.Header>
+            <Expander.ContentTemplate>
+                <DataTemplate>
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto" />
+                            <ColumnDefinition Width="Auto" />
+                        </Grid.ColumnDefinitions>
+                        <Image Source="{Binding ImageUrl}"
+                               Aspect="AspectFill"
+                               HeightRequest="120"
+                               WidthRequest="120" />
+                        <Label Grid.Column="1"
+                               Text="{Binding Details}"
+                               FontAttributes="Italic" />
+                    </Grid>
+                </DataTemplate>
+            </Expander.ContentTemplate>
+    </Expander>
+</Expander>
+```
+
+In this example, tapping the root `Expander` header reveals the header for the child `Expander`:
+
+![Screenshot of an embedded Expander, on iOS and Android (collapsed)](expander-images/embedded-expander1.png "Embedded Expander on iOS and Android (collapsed)")
+
+Tapping the child `Expander` header results in its content being inflated and displayed:
+
+![Screenshot of an embedded Expander, on iOS and Android (expanded)](expander-images/embedded-expander2.png "Embedded Expander on iOS and Android (expanded)")
+
+## Define the expand and collapse animation
+
+The animation that occurs when an `Expander` expands or collapses can be defined by setting the `ExpandAnimationEasing` and `CollapseAnimationEasing` properties to any of the easing functions included in Xamarin.Forms, or custom easing functions. By default, the expand and collapse animations occur over 250ms. However, these durations can be changed by setting the `ExpandAnimationLength` and `CollapseAnimationLength` properties to `uint` values.
+
+The following XAML shows an example of defining the animation that occurs when the `Expander` is expanded or collapsed by the user:
+
+```xaml
+<Expander ExpandAnimationEasing="{x:Static Easing.CubicIn}"
```

Comment:
There's now an `Easing` type converter in Forms 5.0 (https://github.com/xamarin/Xamarin.Forms/pull/11601) so the syntax for specify the easing function can be simplified.

I've just updated the sample to use it.